src/Sentry/FileAttachmentContent.cs

```diff
@@ -8,11 +8,18 @@ namespace Sentry
     public class FileAttachmentContent : IAttachmentContent
     {
         private readonly string _filePath;
+        private readonly bool _readFileAsynchronously;
 
         /// <summary>
         /// Creates a new instance of <see cref="FileAttachmentContent"/>.
         /// </summary>
-        public FileAttachmentContent(string filePath) => _filePath = filePath;
+        /// <param name="filePath">The path to the file to attach.</param>
+        /// <param name="readFileAsynchronously">Whether to use async file I/O to read the file.</param>
+        public FileAttachmentContent(string filePath, bool readFileAsynchronously = true)
```

Comment:
this is a runtime breaking change. eg if someone consuming a 3rd partly dll that uses this api

Reply:
Fixed.  Thanks!